lib/dor/models/describable.rb

```diff
@@ -12,7 +12,7 @@ class CrosswalkError < Exception; end
       has_metadata :name => 'descMetadata', :type => Dor::DescMetadataDS, :label => 'Descriptive Metadata', :control_group => 'M'
     end
 
-    require 'stanford-mods/searchworks'
+    require 'stanford-mods'
```

Comment:
including `stanford-mods/origin_info` (where `pub_year_sort_str` and company live) didn't seem to resolve the method not found issue, but including the whole gem did.  any reason not to?  looks like @atz added that line, so seems like he'd be a good person to object if needed?  https://github.com/sul-dlss/dor-services/commit/4f4d871f20de2b2b51d4024b924752fedb468d21#diff-b031a0a86144a644f8641faccf39e94fR15